test/Microsoft.SqlTools.ServiceLayer.Test/QueryExecution/Execution/ServiceIntegrationTests.cs

```diff
@@ -49,14 +51,23 @@ public class ServiceIntegrationTests
             // ... A completion event should have been fired with empty results
             // ... A batch completion event should have been fired with empty results
             // ... A result set completion event should not have been fired
-            VerifyQueryExecuteCallCount(requestContext, Times.Once(), Times.Once(), Times.Once(), Times.Never(), Times.Never());
+            VerifyQueryExecuteCallCount(requestContext, Times.Once(), Times.Once(), Times.Once(), Times.Once(), Times.Never(), Times.Never());
             Assert.Null(result.Messages);
 
             Assert.Equal(1, completeParams.BatchSummaries.Length);
             Assert.Empty(completeParams.BatchSummaries[0].ResultSetSummaries);
             Assert.NotEmpty(completeParams.BatchSummaries[0].Messages);
 
+            // ... Batch start summary should not contain result sets, messages, but should contain owner URI
+            Assert.NotNull(batchStartParams);
```

Comment:
Minor: you're doing a lot of the same checks in multiple tests (for batchStart, always verify no messages, summaries, but verify URI is set as expected). Could we have some common VerifyBatchStart methods that you call instead? The tests are pretty long and again if you need a new standard property test in the future, you'd update less places.